aries-site/src/examples/templates/forms/managing-child-objects/CollapsedStateAnatomy.js

```diff
@@ -0,0 +1,112 @@
+import { Box, Diagram, Grid, Stack } from 'grommet';
+import { connection } from '../../../../utils';
+import { Annotation } from '../../../../layouts';
+import { ChildHeader } from '../../FormChildObject';
+
+const annotations = {
+  container: { annotation: 'annotation-1', element: 'header-container' },
+  label: { annotation: 'annotation-2', element: 'header-label' },
+  icon: { annotation: 'annotation-3', element: 'header-icon' },
+  valuesSummary: { annotation: 'annotation-4', element: 'header-summary' },
+  border: { annotation: 'annotation-5', element: 'header-container' },
+};
+
+const connections = [
+  connection(
+    annotations.container.annotation,
+    annotations.container.element,
+    'vertical',
+    'rectilinear',
+  ),
+  connection(annotations.label.annotation, annotations.label.element),
+  connection(annotations.icon.annotation, annotations.icon.element),
+  connection(
+    annotations.valuesSummary.annotation,
+    annotations.valuesSummary.element,
+  ),
+  connection(
+    annotations.border.annotation,
+    annotations.border.element,
+    'vertical',
+    'rectilinear',
+  ),
+];
+
+export const CollapsedStateAnatomy = () => {
+  const columns = ['xsmall', 'small', 'small', 'xsmall', 'xsmall'];
+  const rows = ['auto', 'auto', 'auto', 'auto', 'auto'];
+  const areas = [
+    [
+      'empty-1',
+      annotations.container.annotation,
+      annotations.border.annotation,
+      'empty-3',
+      'empty-3',
+    ],
+    [annotations.label.annotation, 'header', 'header', 'empty-4', 'empty-4'],
+    ['empty-5', 'header', 'header', annotations.icon.annotation, 'empty-7'],
+    [
+      annotations.valuesSummary.annotation,
+      'header',
+      'header',
+      'empty-6',
+      'empty-8',
+    ],
+  ];
+
+  return (
+    <Stack interactiveChild="first">
+      <Grid columns={columns} rows={rows} areas={areas} align="center">
+        <Annotation
+          gridArea={annotations.container.annotation}
+          id={annotations.container.annotation}
+          target="1"
+          margin={{ bottom: 'large', horizontal: 'auto' }}
+        />
+        <Annotation
+          gridArea={annotations.label.annotation}
+          id={annotations.label.annotation}
+          target="2"
+        />
+        <Annotation
+          gridArea={annotations.icon.annotation}
+          id={annotations.icon.annotation}
+          target="3"
+          margin={{ left: 'auto' }}
+        />
+        <Annotation
+          gridArea={annotations.valuesSummary.annotation}
+          id={annotations.valuesSummary.annotation}
+          target="4"
+        />
+        <Annotation
+          gridArea={annotations.border.annotation}
+          id={annotations.border.annotation}
+          kind="style"
```

Comment:
why are we making this `purple`? seems like since it's just the `border` doesn't need extra importance on it.

Reply:
This aligns with the direction we took on Layer guidance to differentiate between elements in the anatomy diagram and styles: https://design-system.hpe.design/components/layer?q=layer#anatomy

Reply:
oh instead of `5` then should it be `border="border-weak"`

Reply:
For ease, because the border string was very long (`{{ side: 'top', color: border-weak }}`) Matt decided to just do the number annotation with the purple styling to signal it's referencing a style. Otherwise, the build for the diagram was a bit trickier. Lmk if you're okay with that. I think it follows what we do on the layer for smaller screens.

Reply:
aw okay I see the issue yeah Im okay with that think I wasn't aware of `purple styling to signal it's referencing a style.`